src/paginators/tweet.paginator.v2.ts

```diff
@@ -67,6 +67,11 @@ abstract class TweetTimelineV2Paginator<
   get tweets() {
     return this._realData.data;
   }
+
+  //Includes returned by paginator.
```

Comment:
Actually it may be better to use the jsdoc comment format
```suggestion
  /** Includes returned by paginator */
```